demos/sample_azure_iot_adu/sample_azure_iot_adu.c

```diff
@@ -590,9 +592,9 @@ static AzureIoTResult_t prvEnableImageAndResetDevice()
     for( int32_t ulStepIndex = 0; ulStepIndex < xUpdateResults.ulStepResultsCount; ulStepIndex++ )
     {
         xUpdateResults.pxStepResults[ ulStepIndex ].ulResultCode = 0;
-        xUpdateResults.pxStepResults[ ulStepIndex ].ulExtendedResultCode = 0;
-        xUpdateResults.pxStepResults[ ulStepIndex ].pucResultDetails = NULL;
-        xUpdateResults.pxStepResults[ ulStepIndex ].ulResultDetailsLength = 0;
+        xUpdateResults.pxStepResults[ ulStepIndex ].ulExtendedResultCode = 5678;
```

Comment:
same here

Reply:
That sounds like a good idea. I'll move it to be a macro.